docs/extensions/network.md

```diff
@@ -78,6 +78,12 @@ Once applied, the presumably implemented `Gardenet` extension controller, would
 
 For additional reference, please have a look at the [networking-calico](https://github.com/gardener/gardener-extension-networking-calico) provider extension, which provides more information on how to configure the necessary charts as well as the actuators required to reconcile networking inside the `Shoot` cluster to the desired state.
 
+## Supporting `kube-proxy` less Service Routing
+
+Some networking extensions support service routing without the `kube-proxy` component. This is why gardener supports disabling of `kube-proxy` for service routing by settign `.spec.kubernetes.kubeproxy.enabled` to `false` in the shoot cluster specification. The implicit contract of the flag is that if `kube-proxy` is disabled the networking extension is responsible for the service routing.
```

Comment:
```suggestion
Some networking extensions support service routing without the `kube-proxy` component. This is why Gardener supports disabling of `kube-proxy` for service routing by setting `.spec.kubernetes.kubeproxy.enabled` to `false` in the `Shoot` specification. The implicit contract of the flag is: If `kube-proxy` is disabled then the networking extension is responsible for the service routing.
```

Reply:
Thanks for reading carefully. Will be adapted in the next update.